module-bsp/board/linux/usb_cdc/usb_cdc.cpp

```diff
@@ -220,4 +220,7 @@ namespace bsp
 
         return 0;
     }
+
+    void usbHandleDataReceived()
```

Comment:
means it will no longer work on linux?
I believe it was working previously with virtual tty

Reply:
It will still work there. I haven't changed the implementation there, only changed the API for the stack. Here I added only blank implementation to satisfy the compiler.